src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -862,60 +720,34 @@ private void buildMissingParticipantPanelsForTeam(
         
     }
 
-    private void buildSectionPanelsForMissingSections(Set<String> sectionsWithResponses) {
-        Set<String> sectionsInCourse = bundle.rosterSectionTeamNameTable.keySet();
-        Set<String> sectionsWithoutResponse = new HashSet<String>(sectionsInCourse);
-        sectionsWithoutResponse.removeAll(sectionsWithResponses);
-        
-        List<String> sectionsWithoutResponsesList = new ArrayList<String>(sectionsWithoutResponse);
-        Collections.sort(sectionsWithoutResponsesList);
-        
-        InstructorFeedbackResultsSectionPanel sectionPanel;
-        for (String sectionWithoutResponses : sectionsWithoutResponsesList) {
-            sectionPanel = new InstructorFeedbackResultsSectionPanel();
-            finalizeBuildingSectionPanelWithoutTeamStats(sectionPanel, sectionWithoutResponses);
-            sectionPanels.put(sectionWithoutResponses, sectionPanel);
-            
-            Set<String> teamsInSection = bundle.getTeamsInSectionFromRoster(sectionWithoutResponses);
-            List<String> teamsInSectionAsList = new ArrayList<String>(teamsInSection);
-            
-            Collections.sort(teamsInSectionAsList);
-            
-            for (String teamInMissingSection : teamsInSectionAsList) {
-                List<String> teamMembers = new ArrayList<String>(bundle.getTeamMembersFromRoster(teamInMissingSection));
-                Collections.sort(teamMembers);
-                
-                if (viewType.isPrimaryGroupingOfGiverType()) {
-                    addMissingParticipantsPanelsWithModerationButtonForTeam(
-                                                    sectionPanel, teamInMissingSection, teamMembers);
-                } else {
-                    addMissingParticipantsPanelsWithoutModerationButtonForTeam(
-                                                    sectionPanel, teamInMissingSection, teamMembers);
-                }
-            }
-        }
+    private InstructorFeedbackResultsSectionPanel buildSectionPanel(String name, String nameForDisplay, boolean loadByAjax) {
```

Comment:
Instead of this, let's either have a constructor or factory method for `InstructorFeedbackResultsSectionPanel ` that accepts these 3 parameters.